test/e2e/common/setup.go

```diff
@@ -333,3 +336,113 @@ func SetEnvAndKubeConfigPath(cmd *exec.Cmd, clusterIndex int) {
 	cmd.Env = os.Environ()
 	cmd.Env = append(cmd.Env, fmt.Sprintf("KUBECONFIG=%s/test/%s.kubeconfig", RootDir, ClusterNames[clusterIndex]))
 }
+
+func PatchCoredns(ctx resource.Context) error {
```

Comment:
How about moving it to it's own file `coredns.go`? 

Also.... as we don't have a linter yet... `PatchCoreDNS` :)

Reply:
Some small godoc as to why we are patching could help long term.